near-rs/contract/src/lib.rs

```diff
@@ -37,6 +37,7 @@ pub fn derive_grant_id(grant: &Grant) -> String {
 
 impl Default for FractalRegistry {
     fn default() -> Self {
+        // REVIEWME Let's leave a note on why we started with `g` and not another letter.
```

Comment:
It starts with `g` because that's also the char that the word `grants` starts with.

Are these prefixes something we can leave up to a default? Other than providing you with a convenient footgun for overriding data when you accidentally repeat a prefix, I'm not sure what they're good for or why they live in userland.

Reply:
Happy to take a short dive to see what I can unearth.

Reply:
I feel timed out. Let's just dump what we know, for the next person's benefit?